src/components/dialog/dialog-test.stories.tsx

```diff
@@ -175,3 +176,50 @@ Default.args = {
   disableEscKey: false,
   stickyFooter: false,
 };
+
+export const WithGreyBackground = () => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
**comment**: we may want to set the default to `true`, just so the `Dialog` is open by default when Chromatic sees it 
```suggestion
  const [isOpen, setIsOpen] = useState(true);
```